apps/DroidCam/install

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#check for armhf arch with arm64 kernel
+if [ "$(uname -m)" == "arm64" ] && [ "$(dpkg --print-architecture)" == "armhf" ]; then 
+  warning "You can't install DroidCam on armhf OS with arm64 kernel."
+  exit 1
+fi
+
+libjpegturbo_version=2.1.3
+
+case $arch in
+    32) package_arch=armhf ;;
+    64) package_arch=arm64 ;;
+    *) error "Failed to detect architecture. Something is very wrong." ;;
+esac
+
+git_clone https://github.com/open-sorcerer64/Droid-Cam-RPi || exit 1
+git_clone https://github.com/dev47apps/droidcam || exit 1
+
+# Get dependencies
+install_packages http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator3-1_0.4.92-7_${package_arch}.deb http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator3-dev_0.4.92-7_${package_arch}.deb http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/gir1.2-appindicator3-0.1_0.4.92-7_${package_arch}.deb $HOME/Droid-Cam-RPi/libjpeg-turbo_${libjpegturbo_version}_${package_arch}.deb raspberrypi-kernel-headers libplist-dev libavutil-dev libswscale-dev libswscale-dev libspeex-dev libusbmuxd-dev libasound2-dev make gcc pkg-config libgtk-3-dev adb || exit 1
+
+cd $HOME/droidcam
```

Comment:
```suggestion
mkdir $HOME/droidcam/build
cd $HOME/droidcam/build
```
@cycool29 You didn't make a build folder, so make failed, saying `make[1]: *** /lib/modules/5.10.17-v7l+/build: No such file or directory. Stop.`. This should fix @derryktutt's problem.

Reply:
No. 
The `build` folder is provided with the `raspberrypi-kernel-headers` package. 
An empty `build` folder will indicate another error:

```make
make[1]: Entering directory `/lib/modules/5.10.17-v7l+/build'
make[1]: *** No rule to make target `modules'.  Stop.
make[1]: Leaving directory `/lib/modules/5.10.17-v7l+/build'
make: *** [Makefile:8: all] Error 2

```

Reply:
No. His kernel build folder is missing, not the build folder here.